bin/write_viral_gff.py

```diff
@@ -233,6 +249,7 @@ def empty_if_number(string):
         # coordinates and attributes
         for contig_name, viral_sequence_types in viral_sequences.items():
             clean_contig_name = Record.remove_prophage_from_contig(contig_name)
+            quality = virify_quality[clean_contig_name] if clean_contig_name in virify_quality else "unknown"
```

Comment:
In which case you could have "unknown" quality?

Reply:
there shoud be no cases! That is a "check" for dummy mistake in code